otter_welcome_buddy/startup/database.py

```diff
@@ -8,14 +8,16 @@
 from otter_welcome_buddy.database.models.guild_model import GuildModel
 
 
-async def init_database(bot: Bot) -> None:
+def init_guild_table(bot: Bot) -> None:
+    """Verify that all the guilds that the bot is part of are in the database"""
+    for guild in bot.guilds:
+        print(f"Initializing guild {guild.name} [{guild.id}]")
+        with session_scope() as session:
+            guild_model = GuildModel(id=guild.id)
+            DbGuild.insert_guild(guild_model=guild_model, session=session)
+
```

Comment:
Extra space?

Reply:
Nope, between functions I try to let two spaces to avoid more confusion with python indenting